components/event-publisher-proxy/testing/fixtures.go

```diff
@@ -14,6 +14,7 @@ const (
 
 	MessagingNamespace            = "/messaging.namespace"
 	MessagingEventTypePrefix      = "prefix"
+	JSStreamPrefix                = "sap.kyma.custom"
```

Comment:
as `sap.kyma.custom` is already our event type prefix, would it make sense to use something different for the 
StreamPrefix? 
e.g. `kyma`, `eventing`, ...

Reply:
I agree, changed to `kyma`